Homework

Ildar-Safin opened the pull request:
Сверстал до карточек где начинаются диски.

Ildar-Safin committed:
Homework

Сверстал до карточек где начинаются диски.

Ildar-Safin committed:
add new changes

pocahontassss commented on src/assets/Logotip.svg:
Нужно поработать над наименованием файлов картинок.
https://docs.google.com/document/u/0/d/1ipRxAtW0KEC5bj-vWLNk-p8AwjsN0AWmN8YApyP9VJw/mobilebasic
(Здесь про наименование файлов изображений и как правильно указывать пути к ним)

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,412 @@
+} 
+ 
+.white-color { 
+    color: white; 
```
Цвета всегда в хэше указываем (#$$$$$$) или rgb (255, 255, 255), если нужна прозрачность

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,412 @@
+    box-shadow: 0px 20px 50px 0px #3C3C3B12;
+    max-height: 428px;
+    max-width: 421px;
+    background-image: url(/assets/kraskopult.png);
```
Путь к картинке указываем всегда в кавычках. url("путь к файлу")

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,412 @@
+}
+
+.text-section-shadow-left { 
+    font-family: Inter;
```
Можно вынести в глобальные стили (style.scss), что бы не приходилось каждый раз переписывать данное свойство

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,412 @@
+    line-height: 16.94px;
+}
+
+.shadow-left-link-strelka {
+
+}
```
Пустой класс

pocahontassss commented on src/app/app.component.scss:
```diff
@@ -0,0 +1,412 @@
+.popular-products-kard4 {
+    
+    
+}
```
Пустые классы

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,168 @@
+              <div class="nav-container">
+                  <div class="menu-items-container">
+                      <img src="/assets/Logotip.svg" alt="logotip" style="margin-right: 40px;">
+                      <span class="menu-item white-color">Главная</span>
+                      <span class="menu-item white-color">О компании</span>
+                      <span class="menu-item white-color">Каталог</span>
```
Это у нас должны быть ссылки в оболочке тега nav

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,168 @@
+                  </div>
+                  <div class="header-phone">
+                      <img class="phone" src="/assets/Phone.svg" alt="Phone">
+                      <span>+7 (965) 084-29-29</span>
```
это тоже ссылка с атрибутом href="+tel"

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,168 @@
+              <div class="second-nav-container">
+                  <div class="menu-items-container">
+                   <img src="assets/katalog.svg" alt="katalog" style="margin-right: 10px;">
+                   <span class="menu-item" style="font-size: 14px;">Каталог</span>
+                   <span class="menu-item" style="font-size: 14px;">Стройматериалы</span>
+                   <span class="menu-item" style="font-size: 14px;">Инструменты</span>
+                   <span class="menu-item" style="font-size: 14px;">Электрика</span>
+                   <span class="menu-item" style="font-size: 14px;">Интерьер и одежда</span>
```
это кнопки

pocahontassss commented on src/app/app.component.html:
```diff
@@ -1,336 +1,168 @@
+          <div class="advantage">
+              <img class="advantage-image" src="/assets/big assortiment.svg" alt="brics">
+              <div class="advantage-content">
+                  <div class="advantage-content-title">Большой ассортимент</div>
+                  <div class="advantage-content-text">строительных материалов всегда в наличии
```
это спан и аналогично тоже ниже спаны

Ildar-Safin committed:
add new changes

1. Изменил наименований картинок.
  2. Изменил цвета, в хэше.
  3. Изменил пути, добавил кавычки.
  4. font-family: Inter; вывел в глобальные стили.
  5. Удалил пустые классы.
  6. Обернул в ссылки «tel»  навигатор «Главная, О компании, Каталог»
  7. Добавил кнопки «Каталог, Стройматериалы и тд.»
  8.  «Большой ассортимент, Доставка стройматериалов и Многоуровневая система» обернул в спан.   Телеграмм для связи: @S_ildar_Abd
